hardhat.config.ts

```diff
@@ -0,0 +1,97 @@
+import "@nomicfoundation/hardhat-chai-matchers";
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers";
+import { ethers } from "ethers";
+import fs from "fs";
+import { HardhatUserConfig, task } from "hardhat/config";
+
+require("dotenv").config();
+const DEPLOYER_PRIVATE_KEY = process.env.DEPLOYER_PRIVATE_KEY;
+
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+task("run-script", "Runs a hardhard script by name")
+  .addParam("path", "Path within script/hardhat to script")
+  .setAction(async (taskArgs: { path: string }) => {
+    let main;
+    try {
+      main = require(`./script/hardhat/${taskArgs.path}`);
+    } catch (error) {
+      console.log("Make sure you pass an existing script path. Available scripts:");
+      fs.readdirSync("./script/hardhat", { withFileTypes: true }).forEach((file: fs.Dirent) => {
+        // Some directories don't contain files that can be run this way
+        if (file.isDirectory() && file.name !== "simulations" && file.name !== "utils" && file.name !== "vips") {
```

Comment:
Maybe sth like
```suggestion
        const excludeDirs = ["simulations", "utils", "vips"];
        if (file.isDirectory() && !excludeDirs.includes(file.name)) {
```

but AFAICS there are no such directories in `scripts/`. Maybe it would be ok to omit this condition altogether? 🤔